### DISCUSSION TITLE
Nginx-proxy Incorrect proxy_pass, no live upstreams while connecting to upstream. 502 bad gateway

### DISCUSSION THREAD

**Discussion by duongptryu** [nginx-proxy/nginx-proxy#2156] [Q&A]
I follow step by step to setup nginx-proxy, but when I call API, it's response 502 Bad Gateway. When I saw file default.conf in nginx-proxy container, I saw the proxy_pass line have more a string of numbers after my domain.

This is my command to start nginx-proxy
![Screenshot from 2023-01-30 10-42-57](https://user-images.githubusercontent.com/70091746/215383550-75edeeb9-afa8-4ab3-9f73-4fc9fc29abfa.png)

This is my config of my container app
![Screenshot from 2023-01-30 10-43-30](https://user-images.githubusercontent.com/70091746/215383598-963a52ae-95d3-4a46-988f-64454ac688d8.png)

The error no live upstream because the domain have more string number after.
![Screenshot from 2023-01-30 10-49-46](https://user-images.githubusercontent.com/70091746/215383722-3b9cf845-f15e-4f5e-b87a-4c555ae93de8.png)

The default.conf I saw in nginx-container
![Screenshot from 2023-01-30 10-42-31](https://user-images.githubusercontent.com/70091746/215383796-982e6caf-58ba-41c9-83a9-4a9666e9342d.png)

Please tell my why nginx-proxy automate add more string number after my domain and how I can fix it

**Comment by buchdag** [comment]
Hi.

Could you gave us the whole configurations of both the proxy and the proxied container plus the generated nginx configuration file instead of snippets ?

Also please copy and paste text rather than take screenshots.

**Comment by rhansen** [comment]
Without seeing the full configs it's hard to say, but I'd guess that your nginx-proxy container is not connected to the same network as your proxied container, therefore it can't reach it.
